### PR TITLE
Publish release notes to GitHub

### PR DESCRIPTION
**Pull request by eaba**
* Added publishing to github
* Turned off YAML auto generation
* Deleted `SetDiscription` when packaging nuget

### CODE REVIEW

File: .github/workflows/Windows_release.yml [hunk 1]
```diff
@@ -48,4 +48,5 @@ jobs:
         run: ./build.cmd NuGet
         env:
           Nuget_Key: ${{ secrets.NUGET_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Aaronontheweb) [inline .github/workflows/Windows_release.yml:51]: LGTM

Comment (Aaronontheweb) [reply .github/workflows/Windows_release.yml:51]: This is provided automatically by GitHub Actions, right? Or does the user have to configure something specific for this?

File: build/Build.CI.GitHubActions.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@
 [CustomGitHubActions("pr_validation",
     GitHubActionsImage.WindowsLatest,
     GitHubActionsImage.UbuntuLatest,
-    AutoGenerate = true,
+    AutoGenerate = false,
```
Comment (Aaronontheweb) [inline build/Build.CI.GitHubActions.cs:15]: Good call

File: build/Build.cs [hunk 3]
```diff
@@ -144,35 +146,88 @@ partial class Build : NukeBuild
...
+
+            Information($"GitHub Release {releaseName}");
+            var packages = OutputNuget.GlobFiles("*.nupkg", "*.symbols.nupkg").NotNull();
+            foreach (var artifact in packages)
+            {
+                var releaseAssetUpload = new ReleaseAssetUpload(artifact.Name, "application/zip", File.OpenRead(artifact), null);
+                var releaseAsset = await GitHubClient.Repository.Release.UploadAsset(release, releaseAssetUpload);
+                Information($"  {releaseAsset.BrowserDownloadUrl}");
+            }
+        });
```
Comment (Aaronontheweb) [inline build/Build.cs:230]: LGTM

**Review by Aaronontheweb — APPROVED** [review]
LGTM

File: .github/workflows/Windows_release.yml (hunk 1 above)
Comment (eaba) [reply .github/workflows/Windows_release.yml:51]: It is provided automatically

**Pull request merged**